looprpc/client.proto

```diff
@@ -666,3 +697,124 @@ message LsatToken {
     */
     string storage_name = 8;
 }
+
+message GetLiquidityConfigRequest{}
+
+message LiquidityConfig{
+    /*
+    Set to true if we should include private channels in our balance
+    calculations.
+    */
+    bool include_private = 1;
+
+    /*
+    The target to apply liquidity rules to.
+    */
+    LiquidityTarget target = 2;
+
+    /*
+    A rule to be applied to the target. This value may not be set if setting
+    target to none, and must be supplied otherwise.
+    */
+    LiquidityRule rule = 3;
```

Comment:
Is it necessary to have a separate rule message already? I am worried about trying to design for something that may work out different anyway

Reply:
I think we should design the proto in a way that lets us add more rules easily. I want to add the ability to exclude a peer/channel (requirement for most users I've spoken to) as another rule type. Perhaps we want to add absolute amounts rather than ratios etc.

Reply:
But with the proto structure as is, it doesn't seem to be possible to add another rule?